### PR TITLE
Add bluesky handling

### PR DESCRIPTION
**Pull request by kingthorin**
- kingthorin.md > Add it to my profile (already on psiinon's)
- ricekot.md > Add bluesky to profile.
- list.html > Add it to the contact columns. (bsky.app's icon is pretty lame, I've just gone with a rounded black block for now)
image
image
- single.html > Add it to the info under the pictures.
image

### CODE REVIEW

File: site/layouts/team/single.html [hunk 1]
```diff
@@ -79,6 +79,18 @@ <h1 class="text--white">ZAP Team: {{ .Params.name }}</h1>
               </td>
             </tr>
             {{ end }}
+            {{ if .Params.bluesky }}
+            <tr>
+              <td>
+                <strong>Bluesky</strong>
+              </td>
+              <td>
+                <a rel="me" href="{{ .Params.bluesky }}">
```
Comment (thc202) [inline site/layouts/team/single.html:88]: Is the me functional here too?

Comment (kingthorin) [reply site/layouts/team/single.html:88]: Probably not. I'll remove that, I hadn't compared when I copied the mastodon block.

Thanks

**Comment by kingthorin** [comment]
Removed copy/paste `rel`.

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Add @ricekot : https://bsky.app/profile/ricekot.com ?

**Comment by kingthorin** [comment]
Done

**Review by psiinon — APPROVED** [review]

**Pull request merged**